Nautono/RTOS_framework_v1/src/sail_adc.c

```diff
@@ -19,7 +19,7 @@ static struct adc_module adc;
 // ADC pin mappings
 static const enum adc_positive_input input_pins[ADC_NUM_CHANNELS] = {
```

Comment:
@msohi29 , can add a comment about what each ADC pin is connected to/controlling

Reply:
There is only one ADC pin atm, which is used to get position of rudder.

Reply:
Ok, we should start removing unused code like this. It will be very confusing to others looking at the code